pom.xml

```diff
@@ -18,7 +18,6 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-sftp</artifactId>
-			<version>5.1.0.BUILD-SNAPSHOT</version>
```

Comment:
We don't need this extra dependency in this project at all.
We simply can rely on the parent